web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -199,8 +201,10 @@ export class LayerDialogComponent implements OnDestroy {
 
   onClose() {
     this.dialogRef.close();
-    // TODO: refactor this path into a custom router wrapper
-    return this.router.navigate([`p/${this.projectId}`]);
+    this.ngZone.run(() => {
```

Comment:
Why are Angular zones needed here?

Reply:
I was having a warning here in the console Navigation triggered outside angular zone, that's why used zone here.